### PR TITLE
Make sure we can switch logger whenever we want

### LINKED ISSUE #641: Figure out how to get rid of keyValue for logging
**Describe the bug**
Today we have `keyValue` scattered in different log statements, which are tied to the logging implementation

 **Expected behavior**
We should not be tied to the logging implementation, hence we should remove every statement that couple our code to logstash logback json encoder.

### PR DESCRIPTION
**Pull request by slinkydeveloper**
Signed-off-by: Francesco Guardiani <francescoguard@gmail.com>

Fixes #641

## Proposed Changes

- Just some funky classloader hack to make sure keyValue still works without logstash json encoder in classpath

### CODE REVIEW

File: data-plane/core/src/main/java/dev/knative/eventing/kafka/broker/core/utils/Logging.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
+package dev.knative.eventing.kafka.broker.core.utils;
```
Comment (pierDipi) [inline data-plane/core/src/main/java/dev/knative/eventing/kafka/broker/core/utils/Logging.java:1]: Missing header.

File: data-plane/core/src/test/java/dev/knative/eventing/kafka/broker/core/utils/LoggingTest.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
+package dev.knative.eventing.kafka.broker.core.utils;
```
Comment (pierDipi) [inline data-plane/core/src/test/java/dev/knative/eventing/kafka/broker/core/utils/LoggingTest.java:1]: Missing header.

**Comment by slinkydeveloper** [comment]
@pierDipi do you like the classpath hack? :smile:

**Comment by pierDipi** [comment]
Yes, I like it! 
/lgtm

**Pull request merged**